suite-common/wallet-utils/src/stakingUtils.ts

```diff
@@ -2,17 +2,20 @@ import { fromWei } from 'web3-utils';
 
 import { Account, StakingPoolExtended } from '@suite-common/wallet-types';
 import { BigNumber } from '@trezor/utils/src/bigNumber';
+import { AccountInfo } from '@trezor/connect';
 
-export const getEverstakePool = (account?: Account) => {
-    if (account?.networkType !== 'ethereum') {
-        return undefined;
+export const getEverstakePool = (account?: Account | AccountInfo) => {
```

Comment:
AccountInfo should be enough since it is a subset of `Account`.
```suggestion
export const getEverstakePool = (account?: AccountInfo) => {
```

If you gonna change this, please change it across this whole file.

Reply:
I removed this type change completely, it is not needed.